hackathon/spacecraft/package.json

```diff
@@ -28,7 +28,7 @@
     "@types/jest": "^27.5.1",
     "babel-core": "^7.0.0-bridge.0",
     "babel-jest": "^27.4.6",
-    "expo": "^45.0.4",
+    "expo": "^46.0.16",
```

Comment:
you didn't use ` expo-cli upgrade`

please read https://blog.expo.dev/expo-sdk-46-c2a1655f63f7